### ISSUE TITLE
is apps.py a neccesary file?

### ISSUE THREAD

**Issue by brandonskerritt** [labhackercd/wikilegis#103]
wikilegis/wikilegis/api/apps.py
https://github.com/labhackercd/wikilegis/blob/develop/wikilegis/api/apps.py

This file seems pretty redundant. Is it *really* needed?

**Comment by msfernandes** [comment]
Yes... This file was created with `manage.y startapp` and we forgot to delete :)
